middleware/database/migrations/2023_10_02_162849_create_user_resources_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('user_resources', function (Blueprint $table) {
```

Comment:
Should this be called provider user resource? User is an established concept in Starfish already.